elide-integration-tests/src/test/java/example/ForbiddenBook.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2021, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package example;
+
+import com.yahoo.elide.annotation.Audit;
+import com.yahoo.elide.annotation.Include;
+import com.yahoo.elide.annotation.Paginate;
+import com.yahoo.elide.annotation.ReadPermission;
+
+import javax.persistence.Entity;
+import javax.persistence.Table;
+
+/**
+ * Model for forbidden book.
+ * No Details/Relationships etc.
+ */
+@Entity
+@Table(name = "forbiddenbook")
+@Include
+@Paginate
+@Audit(action = Audit.Action.CREATE,
+        operation = 10,
+        logStatement = "{0}",
+        logExpressions = {"${forbiddenBook.title}"})
+@ReadPermission(expression = "Prefab.Role.None")
```

Comment:
There is already an entity, `NoReadEntity` for testing this kind of security check.